samples/setAutoclass.js

```diff
@@ -14,32 +14,41 @@
  * limitations under the License.
  */
 
-function main(bucketName = 'my-bucket', toggle = false) {
+function main(
+  bucketName = 'my-bucket',
+  toggle = false,
+  terminalStorageClass = 'NEARLINE'
```

Comment:
```suggestion
  toggle = true
  terminalStorageClass = 'ARCHIVE'
```

Apologies for the canonical sample changes, let's update this sample to set toggle = true with terminalStorageClass = 'ARCHIVE'. This is more aligned with the CUJ for setting terminal storage class. Thank you!

Reply:
Done

Reply:
lgtm, thanks Denis!